packages/hooks/package.json

```diff
@@ -11,7 +11,8 @@
     "scripts": {
         "prepack": "yarn clean && tsc",
         "build": "yarn run prepack",
-        "clean": "rm -rf ./dist"
+        "clean": "rm -rf ./dist",
+        "test": "jest"
```

Comment:
Let's add this to GitHub test `Workflows`

Reply:
done